### DISCUSSION TITLE
How to get the 3dtiles file and what are the methods?

### DISCUSSION THREAD

**Discussion by Lshanny** [visgl/deck.gl#6396] [Q&A]
According to the Las file, how to get the 3dtiles file and what are the methods?

**Comment by Pessimistress** [comment]
Check out https://github.com/CesiumGS/3d-tiles/blob/main/ECOSYSTEM.md

**Comment by Lshanny** [comment]
thanks



发自我的iPhone


------------------ Original ------------------
From: Xiaoji Chen ***@***.***&gt;
Date: Sat,Nov 20,2021 4:41 AM
To: visgl/deck.gl ***@***.***&gt;
Cc: Lshanny ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [visgl/deck.gl] How to get the 3dtiles file and what are themethods? (Discussion #6396)





 
Check out https://github.com/CesiumGS/3d-tiles/blob/main/ECOSYSTEM.md
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
